src/init/register_configure_agent.sh

```diff
@@ -13,6 +13,7 @@ INSTALLDIR=${1}
 CONF_FILE="${INSTALLDIR}/etc/ossec.conf"
 TMP_ENROLLMENT="${INSTALLDIR}/tmp/enrollment-configuration"
 TMP_SERVER="${INSTALLDIR}/tmp/server-configuration"
+WAZUH_REGISTRATION_PASSWORD_PATH="${INSTALLDIR}/etc/authd.pass"
```

Comment:
I would rather use the default value ` /etc/authd.pass`, as indicated in the documentation:
https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/client.html#authorization-pass-path

Reply:
2ecd9993ec